aldryn_forms/boilerplates/bootstrap3/templates/admin/aldryn_forms/export_wizard.html

```diff
@@ -1,36 +1,15 @@
-{% extends "admin/change_form.html" %}
+{% extends "admin/aldryn_forms/export.html" %}
 {% load i18n %}
-{% block content %}
-    <h1>{% trans "export form submissions" %}</h1>
-    {% if errors %}
-        <p class="errornote">
-        {% blocktrans count errors|length as counter %}Please correct the error below.{% plural %}Please correct the errors below.{% endblocktrans %}
-        </p>
-        {{ adminform.non_field_errors }}
+
+{% block aldryn_form_hidden %}
+    {{ wizard.management_form }}
+    {{ form.non_field_errors }}
```

Comment:
I don't know what you mean, ``{% block aldryn_form_hidden %}`` is used in export.html directly after the ``<form>``